cmd/test-server/kcp/shard.go

```diff
@@ -105,7 +105,7 @@ func Start(ctx context.Context, name, runtimeDir, logFilePath string, args []str
 
 	go func() {
 		<-ctx.Done()
-		cmd.Process.Kill() // nolint: errcheck
```

Comment:
Not a TODO for you unless you want it, but it's simple enough to

```go
if err := cmd.Process.Kill(); err != nil {
    logger.Error(err, "failed to kill process")
}
```

IMHO no real utility to ignoring errors ...